src/mappings/omim.sssom.tsv

```diff
@@ -1892,7 +1893,7 @@ MONDO:0008863		skos:exactMatch	OMIMPS:210250	Sitosterolemia	semapv:UnspecifiedMa
 MONDO:0008864		skos:exactMatch	OMIM:210350	biemond syndrome 2	semapv:UnspecifiedMatching
 MONDO:0008865		skos:exactMatch	OMIM:210370	bietti crystalline corneoretinal dystrophy	semapv:UnspecifiedMatching
 MONDO:0008866		skos:exactMatch	OMIM:210400	bifid nose, autosomal recessive	semapv:UnspecifiedMatching
-MONDO:0008868		skos:exactMatch	OMIM:210550	biliary malformation with renal tubular insufficiency	semapv:UnspecifiedMatching
+MONDO:0008868		skos:exactMatch	OMIM:210550		semapv:UnspecifiedMatching
```

Comment:
OMIM:210550 was moved to https://www.omim.org/entry/208085

Reply:
Same problem with these. The lexmatch mappings should reflect what is currently in the source, but these mappings come straight from Mondo. It is rather odd to me.